packages/graphiql/test/index.html

```diff
@@ -105,15 +109,16 @@
       }
 
       // Defines a GraphQL fetcher using the fetch API.
+      const graphQLEndpoint = window.location.protocol + '//' + window.location.host + '/graphql';
```

Comment:
Websockets needs the full URL, so I'm being explicit. Note `host` includes the port number if non-standard.